lib/jsdom/living/nodes/SVGElement.idl

```diff
@@ -0,0 +1,6 @@
+interface SVGElement : Element {
```

Comment:
Can you use the interface from https://svgwg.org/svg2-draft/types.html#InterfaceSVGElement instead, commenting out the things that are not implemented yet? Also add a comment at the top of the file like `// https://svgwg.org/svg2-draft/types.html#InterfaceSVGElement`, and don't forget a newline at the end of the file.